.github/workflows/wcfsetup.yml

```
@@ -0,0 +1,26 @@
+name: WCFSetup
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  buildWcfSetup:
+    name: Build WCF setup
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build WCFSetup.tar.gz
+        run: |
+          tar cf com.woltlab.wcf/templates.tar -C com.woltlab.wcf/templates/ --transform 's~^\./~~' .
+          tar cf wcfsetup/install/packages/com.woltlab.wcf.tar -C com.woltlab.wcf --transform 's~^\./~~' --exclude='./templates/*' .
+          tar czf wcfsetup/WCFSetup.tar.gz -C wcfsetup install/ setup/
+          tar czf setup.tar.gz -C wcfsetup WCFSetup.tar.gz install.php test.php
+      - uses: actions/upload-artifact@v2
+        with:
+          name: setup
+          path: setup.tar.gz
```

```suggestion
          path: |
            wcfsetup/WCFSetup.tar.gz
            wcfsetup/install.php
            wcfsetup/test.php
```
Reasoning: GitHub already creates a .zip around this when downloading the artifact.